pkg/cluster/install.go

```diff
@@ -27,10 +27,14 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/version"
 )
 
+func currentTime() time.Time {
+	return time.Now()
+}
+
```

Comment:
Should we follow a similar pattern to `pkg/frontend/frontend.go` -> `now` for consistency? 

```
now func() time.Time